### DISCUSSION TITLE
dolphinscheduler有java的api，完成工作流的创建、任务的创建、工作流的调度、资源文件的实用、参数传递吗？

### DISCUSSION THREAD

**Discussion by LimitAbsolute** [apache/dolphinscheduler#13750] [Q&A]
像https://blog.csdn.net/DolphinScheduler/article/details/127743072里一样，用java来完成 DolphinScheduler 绝大部分核心功能的体验
希望能给一个文档或者实例
谢谢

**Comment by Tandoy** [comment]
There are relevant API documents available at the following address:
https://dolphinscheduler.apache.org/zh-cn/docs/3.1.4/guide/open-api

**Reply by LimitAbsolute** [reply]
我在 https://mvnrepository.com/artifact/org.apache.dolphinscheduler/dolphinscheduler-api 找到了引入maven依赖的方法
<dependency>
            <groupId>org.apache.dolphinscheduler</groupId>
            <artifactId>dolphinscheduler-api</artifactId>
            <version>3.1.4</version>
        </dependency>
这个依赖有调用的例子吗？

**Reply by Tandoy** [reply]
> 我在 https://mvnrepository.com/artifact/org.apache.dolphinscheduler/dolphinscheduler-api 找到了引入maven依赖的方法 org.apache.dolphinscheduler dolphinscheduler-api 3.1.4 这个依赖有调用的例子吗？

I don't think it is, and PyDolphinScheduler should be more appropriate.

**Reply by LimitAbsolute** [reply]
OK, Thanks!
